src/access/PantosRoles.sol

```diff
@@ -7,7 +7,7 @@ pragma solidity 0.8.26;
  *
  * @notice Pantos roles defined as bytes32 constants.
  */
-contract PantosRoles {
+library PantosRoles {
```

Comment:
Just curious on why you decided to use (embedded) library vs contract(abstract)

Reply:
I had a discussion with @markuslevonyak and he suggested that logically it's not an "is a" but much rather a "uses" relationship between the contracts which have a relationship with this contract. So a library would be more appropriate